solutions/secure/README.md

```diff
@@ -0,0 +1,11 @@
+# Cloud Object Storage for IBM Cloud - Financial Services Cloud solution
+
+This architecture creates an Cloud Object Storage(COS) on IBM Cloud®. The solution supports provisioning the following resources:
```

Comment:
First line says `architecture` and second one says `solution`. 
We can say :
This architecture creates an Cloud Object Storage(COS) on IBM Cloud and supports provisioning the following resources: